dts/bindings/riscv/nordic,nrf-vpr-coprocessor.yaml

```diff
@@ -9,14 +9,17 @@ description: |
   VPR is a RISC-V CPU implementation. VPR instances are exposed to other CPUs as
   peripherals.
 
-include: base.yaml
+include: [base.yaml, pinctrl-device.yaml]
 
 properties:
   execution-memory:
     type: phandle
-    required: true
     description: |
-      Memory area from which the VPR core will execute.
+      Memory area from which the VPR code will execute.
+      If not specified, the VPR coprocessor will not be launched automatically
+      by the nordic_vpr_launcher driver when the node is enabled. In such case,
+      the launching is supposed to be done explicitly by the user code, in some
+      custom way presumably.
```

Comment:
I'm curious about this comment. I assume that a custom way of launching VPR boils down to allowing the App core to determine at runtime when and/or where it boots the VPR. Should this really be covered by `zephyr,deferred-init` and maybe an extra driver API for setting the execution address before calling `device_init()`?

Reply:
This is also about a custom way of supplying the code to be executed by the VPR. The application my want to copy the code to RAM by itself and then start the VPR, so in fact do everything that is provided by the nordic_vpr_launcher driver but in its own way.